warehouse/query-core/src/test/java/datawave/query/MultiValueQueryTest.java

```diff
@@ -0,0 +1,93 @@
+package datawave.query;
+
+import datawave.query.testframework.AbstractFunctionalQuery;
+import datawave.query.testframework.AccumuloSetupHelper;
+import datawave.query.testframework.CitiesDataType;
+import datawave.query.testframework.CitiesDataType.CityEntry;
+import datawave.query.testframework.CitiesDataType.CityField;
+import datawave.query.testframework.IDataTypeHadoopConfig;
+import datawave.query.testframework.IFieldConfig;
+import datawave.query.testframework.MultiValueCityFields;
+import org.apache.log4j.Logger;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.Collection;
+
+/**
+ * Performs query test for multivalue fields.
+ * <p>
+ * TODO: Fix composite indexes when this is working.
```

Comment:
Support for composites generated from multi-value fields has been added recently, so you should be able to test for that now.

Reply:
There are specific composite tests that have not been completed yet. Once this pull request is approved I have additional changes to the framework that are needed to support some of the features that are required for the uncompleted tests. I was intending to address any issues related to composites at that time.